python/feathub/registries/mysql_registry.py

```diff
@@ -0,0 +1,273 @@
+    ) -> List[TableDescriptor]:
+        result = []
+        for table in feature_descriptors:
+            if table.name == "":
+                raise FeathubException(
+                    "Cannot build a TableDescriptor with empty name."
+                )
+            self.tables[table.name] = (
+                table,
+                table.build(self, force_update=force_update, props=props),
+            )
+            result.append(self.tables[table.name][1])
+
+        return result
+
+    def register_features(
+        self, feature_descriptors: List[TableDescriptor], force_update: bool = False
+    ) -> bool:
+        self.build_features(feature_descriptors, force_update=force_update)
+
+        for descriptor in feature_descriptors:
+            existing_features = self._get_feature_from_remote_registry_or_none(
+                descriptor.name
+            )
+            if (
+                existing_features is not None
+                and existing_features[1].to_json()
+                == self.tables[descriptor.name][1].to_json()
+            ):
+                continue
+
+            original_json = json.dumps(
+                self.tables[descriptor.name][0].to_json(), sort_keys=True
+            ).replace('"', '\\"')
+            built_json = json.dumps(
+                self.tables[descriptor.name][1].to_json(), sort_keys=True
+            )
+            digest = _get_digest(built_json)
+            built_json = built_json.replace('"', '\\"')
+            self.cursor.execute(
+                f"""
+                    INSERT INTO {self.table} (
+                       `digest`,
+                       `name`,
+                       `timestamp`,
+                       `is_deleted`,
+                       `original_json`,
+                       `built_json`
+                    ) VALUES (
+                        "{digest}",
+                        "{descriptor.name}",
+                        NOW(),
+                        False,
+                        "{original_json}",
+                        "{built_json}"
+                    );
+                """
+            )
+        return True
+
+    def get_features(
+        self, name: str, force_update: bool = False, is_built: bool = True
+    ) -> TableDescriptor:
+        if not force_update and name in self.tables:
+            return self.tables[name][1 if is_built else 0]
+
+        existing_feature = self._get_feature_from_remote_registry_or_none(name)
+        if existing_feature is None:
+            raise RuntimeError(
+                f"Table '{name}' is not found in the cache or registry. "
+                "Please invoke build_features(..) for this table."
+            )
+
+        self.tables[name] = existing_feature
+        return self.tables[name][1 if is_built else 0]
+
+    def delete_features(self, name: str) -> bool:
+        self.cursor.execute(
+            f"""
+                UPDATE {self.table}
+                SET `is_deleted`=True
```

Comment:
I am not sure we need `is_delete` field.

Would it be simpler to just delete the corresponding row from mysql?